setup/setup.iss

```diff
@@ -21,14 +23,16 @@ SolidCompression=yes
 OutputDir=build
 SetupLogging=yes
 ChangesAssociations=yes
+AllowNoIcons=yes
+AllowRootDirectory=yes

```

Comment:
why enabling `AllowRootDirectory`? Installing Dash Faction in a drive root sounds like a bad idea

Reply:
You're right. I initially saw this as something I'd do to have external drives dedicated to different versions of Dash, but it's not very practical compared to just installing to a folder in the drive. I'll revert it.